web/components/App.jsx

```diff
@@ -65,47 +87,32 @@ class App extends React.Component {
       const newValue = this.state.value;
       newValue[role.label] = values;
       this.setState({ value: newValue });
+
+      values = (values ? values : '');
+      const outputValue = (values.label ? values.label : '');
+      const eventMessage = (outputValue.length ?
```

Comment:
Do we need the parentheses?

Reply:
its not required, just a convention to make the ternary operation more readable as it has sequential operations in one single line and can have nested operations.

Reply:
Also to avoid mistakes as the ternary operator has very low precedence, lower than operations /, +, <, etc; it is also lower than the precedence of the minus operator. As a result, the program doesn’t work in the way we expected.

Reply:
but really is a habit, I am ok to remove it

Reply:
actually the line 93 was already like that, so I think it's already been used

Reply:
Then it's perfect fine.